cernopendata/modules/fixtures/data/records/cms-primary-datasets.json

```diff
@@ -121,7 +121,7 @@
       "attribution": "CC0"
     },
     "methodology": {
-      "description": "Events stored in this primary dataset were selected because of presence of more than one isolated tau in the event, or because of presence of low-energy muons from b-quark decay."
+      "description": "Events stored in this primary dataset were selected because of presence of more than one isolated tau in the event, or because of presence of low-energy muons from b-quark decay.       <p><strong>HLT trigger paths</strong>      <br/>The possible HLT trigger paths in this dataset are:      <br/>HLT_BTagIP_Jet50U      <br/>HLT_BTagMu_DiJet10U      <br/>HLT_BTagMu_DiJet20U      <br/>HLT_BTagMu_DiJet20U_Mu5      <br/>HLT_BTagMu_DiJet30U      <br/>HLT_BTagMu_DiJet30U_Mu5      <br/>HLT_BTagMu_Jet10U      <br/>HLT_BTagMu_Jet20U      <br/>HLT_DoubleIsoTau15_OneLeg_Trk5      <br/>HLT_DoubleIsoTau15_Trk5      <br/>HLT_DoubleLooseIsoTau15      <br/>HLT_SingleIsoTau20_Trk15_MET20      <br/>HLT_SingleIsoTau20_Trk15_MET25      <br/>HLT_SingleIsoTau20_Trk5      <br/>HLT_SingleIsoTau20_Trk5_MET20      <br/>HLT_SingleIsoTau30_Trk5      <br/>HLT_SingleIsoTau30_Trk5_L120or30      <br/>HLT_SingleIsoTau30_Trk5_MET20      <br/>HLT_SingleIsoTau35_Trk15_MET25      <br/>HLT_SingleLooseIsoTau20      <br/>HLT_SingleLooseIsoTau25_Trk5</p>"
```

Comment:
Can you please add `<p>..</p>` to the leading text ("Events stored...") so that the whole `methodology.description` field would be following paragraph markup?

Reply:
Thank you, fixed!